ParkingZoneApp/Program.cs

```diff
@@ -11,12 +14,17 @@ public static void Main(string[] args)
             var builder = WebApplication.CreateBuilder(args);
 
             // Add services to the container.
-            var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
+            var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") 
+                ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
+
             builder.Services.AddDbContext<ApplicationDbContext>(options =>
                 options.UseSqlServer(connectionString));
+
+            builder.Services.AddScoped(typeof(IRepository<>), typeof(Repository<>));
```

Comment:
I think project should run without this line. lets remove it

Reply:
indeed